docs/concepts.rst

```diff
@@ -32,3 +32,53 @@ Variable
 A *variable* represents a process measure of one :ref:`appliance <appliance>` of which
 timeseries data is captured and stored in the ENLYZE platform. One appliance may have
 many variables, whereas one variable is only associated with one appliance.
+
+
+.. _production_order:
+
+Production Order
+----------------
+
+A *production order* represents the goal of producing a certain quantity of a
+given :ref:`product <product>`. Production orders are usually created in an ERP
+system or MES of the customer and then synchronized into the ENLYZE platform.
+They are referenced by an identifier which oftentimes is a short combination of
+numbers and/or characters, like FA23000123 or 332554.
+
+In the ENLYZE platform, a production order always encompasses the production of
+one single :ref:`product <product>` on one single :ref:`appliance <appliance>`.
+
+
+.. _production_run:
+
+Production Run
+--------------
+
+A *production run* represents the real allocation of a :ref:`production order
+<production_order>` on an :ref:`appliance <appliance>`. Usually, the operator of
+the appliance uses some kind of interface to log the time when a certain
+production order has been worked on. A production order oftentimes is not
```

Comment:
:+1:

Reply:
What about this?

```suggestion
the appliance uses an interface to log the time when a certain
production order has been worked on. For instance, this could be the appliance's HMI or a tablet computer next to it. A production order oftentimes is not
```